### DISCUSSION TITLE
PromCon North America 2021

### DISCUSSION THREAD

**Discussion by roidelapluie** [prometheus/prometheus#9375] [Announcements]
PromCon North America 2021 is the seventh installment of the PromCon conference dedicated to the Prometheus monitoring system. The conference will be presented as a hybrid event - attendees can join and participate in person or virtually. It will take place on Monday, October 11, 2021, as a colocated event at KubeCon + CloudNativeCon North America.

https://promcon.io/2021-losangeles/

**Comment by LeviHarrison** [comment]
Talks have been posted on YouTube: https://youtube.com/playlist?list=PLj6h78yzYM2M6ebZc9c-AxNvcg91lG18L
